lib/gui/os/window-progress.js

```diff
@@ -38,12 +38,15 @@ exports.currentWindow = electron.remote.getCurrentWindow();
  * Show progress inline in operating system task bar
  *
  * @param {Number} percentage - percentage
+ * @param {String} status - flashing/validating status
  *
  * @example
  * windowProgress.set(85);
  */
-exports.set = (percentage) => {
+exports.set = (percentage, status) => {
   exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
+  const statusString = status === 'check' ? 'Validating' : 'Flashing';
```

Comment:
For now yes. If it ever changes I left the argument `status` as a string instead of using a boolean so we can just update this function to check for other things.